src/test/java/de/internetsicherheit/brl/bloxberg/cache/ArgParserTest.java

```diff
@@ -0,0 +1,37 @@
+package de.internetsicherheit.brl.bloxberg.cache;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+
+public class ArgParserTest {
+    @Test
+    void testParser() throws IOException {
+        ArgParser parser = new ArgParser();
+
+        String[] args = new String[5];
+        // defaults
```

Comment:
These are not the defaults, quite the contrary, what does this comment indicate?
Please remove the comment if invalid (nothing worse than wrong comments).